rlclientlib/serialization/fb_serializer.h

```diff
@@ -242,6 +248,33 @@ namespace reinforcement_learning { namespace logger {
     buffer_t& _buffer;
   };
 
+  template <>
+  struct fb_event_serializer<generic_event> {
+    using fb_event_t = v2::Event;
+    using offset_vector_t = typename std::vector<flatbuffers::Offset<fb_event_t>>;
+    using batch_builder_t = v2::EventBatchBuilder;
+
+    static size_t size_estimate(const generic_event& evt) {
+      return evt.get_payload().size();
+    }
+
+    static int serialize(generic_event& evt, flatbuffers::FlatBufferBuilder& builder,
```

Comment:
why not add a simple unit test for all the new serializers?

Reply:
good idea, working on it.

Reply:
done